box/src/Box.module.css

```diff
@@ -1,11 +1,11 @@
 .wrapper {
   position: relative;
   z-index: 0;
+  overflow: hidden;
 }
 
 .checkered {
-  opacity: 1;
-  background-color: #000000;
+  opacity: 1; /* needed to avoid override by showcasing classes and styles*/
```

Comment:
despite the comment, I'm not sure I understand it's still needed
I was debugging and saw it worked without it